torch/csrc/jit/codegen/cuda/scheduler/reduction.cpp

```diff
@@ -534,9 +538,9 @@ ReductionParams OuterReductionHeuristic(
   // domain for this
 
   // Blocks for reductions
-  int64_t gdimy = 1;
+  int64_t grdim = 1;
```

Comment:
What do the names of `grdim` and `gidim` mean?

Reply:
grid dim in the reduction and grid dim in the iter, will add a comment.